kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -421,7 +421,7 @@ def foundry_prove(
     break_on_jumpi: bool = False,
     break_on_calls: bool = True,
     implication_every_block: bool = True,
-    bmc_depth: int | None = None,
+    bmc_depth: int | None = 3,
```

Comment:
I still wouldn't change this (and the one in `_method_to_apr_proof`) to avoid breaking callers. If you do, make sure to adjust all calls to the function (in particular in `test_foundry_prove.py`).

Reply:
Thanks for the explanation! I've changed the default `bmc_depth` value to `None` in all functions (https://github.com/runtimeverification/evm-semantics/pull/1963/commits/69082952fda2f053a132dbe9a654d75e5d46e832). You're right, having `3` as a default  value for `--bmc-depth` in CLI is sufficient.